Adds show/hide to rule example (#113)
eclipsesource/jsonforms2-website: src/pages/examples/rule.mdx

# Rule

> This example uses two [rules](/docs/uischema/rules). The first rule controls the enablement of a control if the 'Is Alive' checkbox is unchecked.

Reviewer: Needs to be updated because of the example restructuring
```suggestion
This example uses two [rules](/docs/uischema/rules). The first rule manages the enablement of a control based on whether the 'Is Dead' checkbox is checked.
```

> The second rule controls the visability of a control if the "Eats vegetables?" checkbox is checked.

Reviewer: It's `visibility` ;)
```suggestion
The second rule manages the visibility of a control based on whether the "Eats vegetables?" checkbox is checked.
```
